components/user/cover-image/cover-image.js

```diff
@@ -1,7 +1,7 @@
 import Placeholder from '@components/generic/loading/loading-placeholder/loading-placeholder'
 import loadingPlaceholder from '../../generic/loading/loading-placeholder/loading-placeholder'
 
-export default function ({ user, loading = false }) {
+export default function Named({ user, loading = false }) {
```

Comment:
@ihorkitrum  instead of naming them "Named" can you put the name of the component. for ex: CoverImage

Reply:
I did this to quickly clear the console warnings. So in future we'll change all this function names to correct ones